src/controllers/person/index.js

```diff
@@ -0,0 +1,55 @@
+import { isEmpty } from 'lodash'
+
+export const getAddress = (obj) => obj.getPhysicalAddress()
+
+export const getPersonalInformation = async (obj, { userId }, { models }) => {
+  const person = await models.person.findOne({ where: { userId } })
+
+  if (isEmpty(person)) throw new Error('Provided UserId could not be found in the database.')
+
+  return person
+}
+
+export const setPersonalInformation = (obj, { userId, personalInformation }, { models }) => {
```

Comment:
I don't like the implementation of this function (although it will probably be removed and its implementation moved elsewhere when you fix the schema). Maybe we should split it into different methods in the `Person` model that, for example, will update a whole person including address information at once. For example: `person.updateWithAssociations({ ...personFields, physicalAddress: addressFields }, someSequelizeOptions)`